smallrye-reactive-messaging-kafka/pom.xml

```diff
@@ -36,6 +36,22 @@
       <version>${kafka.version}</version>
     </dependency>
 
+    <dependency>
```

Comment:
Is there the ability to use SmallRye OpenTracing, or we need OpenTracing directly?

Reply:
It would pull it other unnecessary dependencies (jax-rs, cdi instrumentations namely).